mobify/astro-scaffold
# initMainLayout During Welcome Screen Show

Previsouly the main layout would be inited after the welcome modal had disappeared

```javascript
welcomeModalController
    .show({forced: true})
    .finally(initMainLayout);
```

With this Astro change, https://github.com/mobify/astro/pull/684, the main layout can be inited while the welcome modal is showing. This is ideal because when you dismiss the loader the content will already be there and ready to go.

```javascript
welcomeModalController.show({forced: true});
initMainLayout();
```

JIRA: N/A
Linked PRs: https://github.com/mobify/astro/pull/684

## Changes
- `initMainLayout` after calling show welcome modal instead of after it hides

## How to test-drive this PR
- Point `astro-sdk` dependency to `main-view-plugin-bottom` and run `npm run deps`
- Run app on iOS and Android
- When the welcome modal shows, wait a few seconds so the main content can load (makes the change easier to see)
- Dismiss the welcome modal, the content should be loaded and ready to go

## TODOS:
- [ ] https://github.com/mobify/astro/pull/684 has landed 🚢
